src/components/LandingPage/LandingPage.css

```diff
@@ -0,0 +1,25 @@
+.landing {
+    display: flex;
+    flex-direction: column;
+    border: 3px solid black; /* temporary to visualize component */
+    width: 425px; /* mobile size placeholder */
+    height: 725px; /* mobile size placeholder */
```

Comment:
I'm realizing now that I have a propensity to use grid for everything, and it seems like you usually use flex (unless something is literally a grid I assume). I think this looks really good and I think I'll try to style more in flex since sometimes they can act weird if you get them together.

Reply:
Yeah I usually reach for grid for things like idea boxes but I think we can have a healthy combo of both! Not sure if there's a right or wrong way to do it.